lib/cli.js

```diff
@@ -439,7 +440,11 @@ internals.options = function () {
         options.output = process.stdout;
     }
 
-    if (options.assert) {
+    if (options.assert && options.assert.toLowerCase() === 'false') {
```

Comment:
Maybe combine the `if` and `else if` into a single `if (options.assert)`, then branch inside there.